cpp/harvest/nacjd.cc

```diff
@@ -158,6 +165,29 @@ void ExtractExistingIDsFromMarc(MARC::Reader * const marc_reader, std::set<std::
     }
 }
 
+void ExtractOneIDFromWebsite(const unsigned marc_id) {
+    if (FileUtil::Exists(NACJD_TITLES))
```

Comment:
The paths NACJD_TITLES and NACJD_NEW_TITLES_JSON should not be hardcoded. There should be an `output_path` variable that should be parsed from argv and passed to this function (and also: for this function we only need a single output_path instead of 2).